apps/flux/lib/flux/event_listener.ex

```diff
@@ -49,6 +50,15 @@ defmodule Flux.EventListener do
     {:noreply, state}
   end
 
+  def handle_info({:cast_event, :price_broadcast, price}, state) do
+    Logger.info("[F] Processing Price: #{inspect(price)}")
+    IO.puts("PBF")
```

Comment:
Already logging the payload so maybe we should drop the `IO.puts`?

Reply:
Agree let's drop `IO.puts`